tools/test.py

```diff
@@ -106,6 +107,8 @@ def main():
         distributed = True
         init_dist(args.launcher, **cfg.dist_params)
 
+    # create work_dir
+    mmcv.mkdir_or_exist(osp.abspath(cfg.work_dir))
```

Comment:
It seems redundant to `mkdir` for test

Reply:
If output_config is given, like [here](https://github.com/open-mmlab/mmaction2/blob/master/configs/localization/bmn/bmn_400x100_2x8_9e_activitynet_feature.py#L104), in [line 144](https://github.com/open-mmlab/mmaction2/blob/master/tools/test.py#L144) it will save to file, and it will not check if the dir is already existed.
So if someone have never run the training script (I think it is common to download a ckpt and have a test in first after clone a repo), it will throw out an error after several minutes of model inference.

Reply:
maybe this check should be added into mmcv file dump